src/digitKeyboard.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+import { EventDispatcher } from './eventDispatcher.js';
+
+/**
+ *  DigitKeyboard sets the digits on the screen.
+ */
+export class DigitKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindEvents();
+    }
+
+    /**
+     * Triggers binded event with a given value.
+     */
+    _bindEvents() {
+        let buttons = document.querySelector('.allButtons');
+        let numberButtons = buttons.querySelectorAll('.number');
+
+        for (let i = 0; i < numberButtons.length; i++) {
+            let self = this;
+            numberButtons[i].addEventListener("click", function() {
+                let buttonDigit = this.innerHTML;
+                self.trigger('DigitKeyboard.KEY_PRESSED', buttonDigit);
```

Comment:
You can bind a function to a context, so investigate function binding (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind). Using it, we should have here:
`this.trigger(...)`